modules/sqs/main.tf

```diff
@@ -0,0 +1,74 @@
+## Terraform Requirements
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"
+    }
+  }
+}
+
+
+## AWS Provider Settings
+provider "aws" {
+  region = var.region
+}
+
+## Local Variables
+locals {
+  tags = merge(var.tags, { Deployment = var.prefix })
+}
+
+## copy_files_to_archive_get_message_queue - SQS queue that copy_files_to_archive lambda writes to
```

Comment:
Doesn't copy_files_to_archive read from this queue to get what it needs to copy next instead of writing? I thought this Lambda would only write to the database status update queue.